app/assets/stylesheets/application/modules/_highlight.scss

```diff
@@ -4,48 +4,37 @@ pre {
   background-color: #f0f0f5;
 }
 
+.CodeMirror {
+  line-height: 1.3;
+  font-family: 'Monaco', 'Menlo', 'Ubuntu Mono', 'Consolas', 'source-code-pro', monospace;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+}
+
 /* identifier */
-.na, .nb, .nc, .no, .nd, .ni, .bp, .vc, .vg, .vi, .nx, .nv,
-.nn, .nt,
-.ace_function,
-.ace_variable,
-.ace_parameter,
-.ace_identifier,
-.ne,
-.nf,
-.nl {
+.cm-def {
   color: black !important;
 }
 
 /* keyword */
-.k, .kn, .kp, .kr, .kt, .kd, .ace_keyword, .ace_type {
+.cm-keyword {
   color: $brand-info  !important;
   font-weight: bold !important;
 }
 
 /* literal */
-.m, .mf, .mh, .mi, .mo,
-.s, .sb, .sc, .sd, .s2, .se, .sh, .sx, .sr, .s1, .ss,
-.kc,
-.il,
-.l,
-.ace_boolean, .ace_numeric,
-.ace_string, .ace_symbol,
-.ace_hexadecimal, .ace_character,
-.ace_decimal {
+.cm-number {
   color: #369434 !important;
 }
 
 /* operator */
-.ow, .o,
-.ace_operator,
-.ace_meta.ace_rule.ace_definition.ace_prolog {
+.cm-operator {
   color: $brand-complementary !important;
   font-weight: bold !important;
 }
 
 /* comment */
-.ace_comment, .c, .cm, .cp, .c1, .cs {
+.cm-comment {
```

Comment:
I think we add code mirror clases without remove ace-editor and other clases. And make styles backward compatible.

However, I think this styles are duplicated in mumuki/mumuki-styles